server/lib/settings/index.ts

```diff
@@ -78,6 +81,10 @@ export interface DVRSettings {
   tagRequests: boolean;
 }
 
+export interface TvdbSettings {
+  use: boolean;
+}
```

Comment:
What's the point of this `use` boolean inside of the `tvdb` setting? Why not just a `tvdb: boolean` setting?

Reply:
That's right, I'll make the change to make it simpler.

Reply:
I've replaced the value in settings with a boolean